Tests/Form/Type/DateTimePickerTypeTest.php

```diff
@@ -24,7 +24,7 @@ class DateTimePickerTypeTest extends \PHPUnit_Framework_TestCase
 {
     public function testGetName()
     {
-        $type = new DateTimePickerType(new MomentFormatConverter());
+        $type = new DateTimePickerType(new MomentFormatConverter(), $this->getMock('Symfony\Component\Translation\TranslatorInterface'));
```

Comment:
https://github.com/sonata-project/SonataCoreBundle/commit/2b89a9f33eb9080f063a06dfb6195f466006a4eb : Am I supposed to create a test which is catching the deprecated call ?